flink-connector-bigquery/src/test/java/com/google/cloud/flink/bigquery/fakes/StorageClientFaker.java

```diff
@@ -81,7 +81,7 @@ public QueryDataClient getQueryDataClient(CredentialsOptions readOptions) {
                 @Override
                 public List<String> retrieveTablePartitions(
                         String project, String dataset, String table) {
-                    return new ArrayList<>();
```

Comment:
Why do we want to increase out guava dependency, especially where there's no clear advantage in this case? can we revert?

Reply:
done.